alacritty_terminal/src/term/cell.rs

```diff
@@ -92,6 +93,20 @@ impl Cell {
         self.extra.get_or_insert_with(Default::default).zerowidth.push(c);
     }
 
+    #[inline]
+    pub fn ul(&self) -> Option<Color> {
```

Comment:
Please don't use abbreviations like `ul` and instead spell it out. This applies to all changes in this PR.

Reply:
I followed the convention set by `fg` and `bg` here. I guess `ul` is less obious though

Reply:
Yeah, fg/bg could be renamed too, but they're **really** obvious. Nobody knows what `ul` is. Generally I would encourage using the full length as much as possible.

Reply:
Yes, I agree